codegen/test/rtc/src/compile_kernel.cpp

```diff
@@ -89,6 +91,12 @@ kernel compile_kernel(const std::vector<src_file>& srcs, compile_options options
 
     auto obj = read_buffer(out_path.string());
 
+    std::ofstream ofh("obj.o", std::ios::binary);
+    for(auto i : obj)
+        ofh << i;
+    ofh.close();
+    // int s = std::system(("/usr/bin/cp " + out_path.string() + " codeobj.bin").c_str());
```

Comment:
Should it be removed?

Reply:
I left that in the case I need to dump the code object in ELF format. What is the convention to leave debug code?